include/SDL3/SDL_gpu.h

```diff
@@ -159,6 +159,24 @@
  * and reasonable to implement across several platforms and underlying APIs.
  * So while these things are not in the "never" category, they are definitely
  * not "near future" items either.
+ *
+ * ## System Requirements
+ *
+ * **Vulkan:** Vulkan 1.0 with the following extensions and device features:
+ * - `VK_KHR_swapchain`
```

Comment:
I wonder if it's worth fussing with this for windowless rendering, probably better to wait for the wildly unlikely scenario where this doesn't exist?

Reply:
We can investigate windowless rendering in the future, but currently this is a hard requirement in the Vulkan driver, so even though it's obvious I think it's worth mentioning.